### ISSUE TITLE
[RuntimeException]   curl_multi_select failure

### ISSUE THREAD

**Issue by arthurkirkosa** [hirak/prestissimo#64]
```
 Prefetch start: total: 17


  [RuntimeException]
  curl_multi_select failure


Exception trace:
 () at /Users/arthurkirkosa/.composer/vendor/hirak/prestissimo/src/CurlMulti.php:112
 Hirak\Prestissimo\CurlMulti->wait() at /Users/arthurkirkosa/.composer/vendor/hirak/prestissimo/src/ParallelDownloader.php:54
 Hirak\Prestissimo\ParallelDownloader->download() at /Users/arthurkirkosa/.composer/vendor/hirak/prestissimo/src/Plugin.php:115
 Hirak\Prestissimo\Plugin->onPostDependenciesSolving() at n/a:n/a
 call_user_func() at phar:///Users/arthurkirkosa/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:166
 Composer\EventDispatcher\EventDispatcher->doDispatch() at phar:///Users/arthurkirkosa/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:133
 Composer\EventDispatcher\EventDispatcher->dispatchInstallerEvent() at phar:///Users/arthurkirkosa/bin/composer/src/Composer/Installer.php:516
 Composer\Installer->doInstall() at phar:///Users/arthurkirkosa/bin/composer/src/Composer/Installer.php:219
 Composer\Installer->run() at phar:///Users/arthurkirkosa/bin/composer/src/Composer/Command/UpdateCommand.php:173
 Composer\Command\UpdateCommand->execute() at phar:///Users/arthurkirkosa/bin/composer/vendor/symfony/console/Command/Command.php:259
 Symfony\Component\Console\Command\Command->run() at phar:///Users/arthurkirkosa/bin/composer/vendor/symfony/console/Application.php:844
 Symfony\Component\Console\Application->doRunCommand() at phar:///Users/arthurkirkosa/bin/composer/vendor/symfony/console/Application.php:192
 Symfony\Component\Console\Application->doRun() at phar:///Users/arthurkirkosa/bin/composer/src/Composer/Console/Application.php:166
 Composer\Console\Application->doRun() at phar:///Users/arthurkirkosa/bin/composer/vendor/symfony/console/Application.php:123
 Symfony\Component\Console\Application->run() at phar:///Users/arthurkirkosa/bin/composer/src/Composer/Console/Application.php:99
 Composer\Console\Application->run() at phar:///Users/arthurkirkosa/bin/composer/bin/composer:43
 require() at /Users/arthurkirkosa/bin/composer:25
```

The version I'm using is *hirak/prestissimo                  0.1.14* and the error appeared today, after doing a composer global update

I'm on a mac

**Comment by arthurkirkosa** [comment]
Working fine with 0.1.13. Just tested it.

**Comment by DracoBlue** [comment]
Same here.

**Comment by DracoBlue** [comment]
Fixed with #66
